service/awsconfig/v5/resource/loadbalancer/delete.go

```diff
@@ -2,13 +2,32 @@ package loadbalancer
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
 )
 
-// TODO
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+	deleteInput, err := toLoadBalancerState(deleteChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if deleteInput != nil && len(deleteInput.LoadBalancerNames) > 0 {
+		for _, lbName := range deleteInput.LoadBalancerNames {
+			if _, err := r.clients.ELB.DeleteLoadBalancer(&elb.DeleteLoadBalancerInput{
+				LoadBalancerName: aws.String(lbName),
+			}); err != nil {
+				return microerror.Mask(err)
+			}
+		}
+	}
+
+	r.logger.LogCtx(ctx, "debug", fmt.Sprintf("deleted %d load balancers", len(deleteInput.LoadBalancerNames)))
+
```

Comment:
You may want to apply log conventions like here:
https://github.com/giantswarm/azure-operator/blob/25f344e74cb1617f0443849e3f2bd2c53263b1b0/service/resource/resourcegroup/resource.go#L172-L194